config/routes/api.rb

```diff
@@ -121,6 +121,10 @@
       resource :activity, only: [:show], controller: 'instances/activity'
     end
 
+    namespace :peers do
+      get :search, to: 'search#index'
```

Comment:
Shouldn't search be a POST request?

Reply:
I checked and other search APIs we have are GET